### PR TITLE
[python-0326] Implement solution

### PR DESCRIPTION
**Pull request by CC972**
Tried to generalise problem to work with any positive integer base using a function factory...

### CODE REVIEW

File: python/tests/test_problem_0326_power_of_three.py [hunk 1]
```diff
@@ -0,0 +1,225 @@
+import pytest
+from solutions.problem_0326_power_of_three import is_power_of
+
+
+@pytest.fixture(scope="session")
+def power_of_three():
```
Comment (CC972) [inline python/tests/test_problem_0326_power_of_three.py:6]: Ideally I want to have this setup fixture inside each test class instead (as can be done using unittest). Any suggestions on how to do this?

**Review by GreenSlime96 — CHANGES_REQUESTED** [review]

File: python/tests/test_problem_0326_power_of_three.py [hunk 2]
```diff
@@ -0,0 +1,225 @@
...
+@pytest.fixture(scope="session")
+def power_of_twenty_one():
+
+    yield is_power_of(21)
+
+
+class TestPowerOfThree:
+
+    @pytest.fixture(autouse=True)
+    def setup(self, power_of_three):
```
[the remark this answers is not in the document]
Comment (CC972) [reply python/tests/test_problem_0326_power_of_three.py:26]: That's great! Works inside the class now.

File: python/solutions/problem_0326_power_of_three.py [hunk 3]
```diff
@@ -0,0 +1,23 @@
+from typing import Callable
+
+
+def is_power_of(base: int) -> Callable:
```
[the remark this answers is not in the document]
Comment (CC972) [reply python/solutions/problem_0326_power_of_three.py:4]: Ah nice I didn't know you could type hint the arguments and return type too :D

**Review by GreenSlime96 — CHANGES_REQUESTED** [review]

**Review by GreenSlime96 — APPROVED** [review]
Well done Chiguana

**Pull request merged**